src/SprykerEco/Shared/ComputopApi/Transfer/computop.transfer.xml

```diff
@@ -4,119 +4,119 @@
-        <property name="payId" type="string" />
-        <property name="transId" type="string" />
-        <property name="amount" type="int" />
-        <property name="currency" type="string" />
-        <property name="capture" type="string" />
-        <property name="mac" type="string" />
-        <property name="reqId" type="string" />
-        <property name="refNr" type="string" />
-        <property name="orderDesc" type="string" />
-        <property name="date" type="string" />
+        <property name="merchantId" type="string"/>
+        <property name="payId" type="string"/>
+        <property name="transId" type="string"/>
+        <property name="amount" type="int"/>
+        <property name="currency" type="string"/>
+        <property name="capture" type="string"/>
+        <property name="mac" type="string"/>
+        <property name="reqId" type="string"/>
+        <property name="refNr" type="string"/>
+        <property name="orderDesc" type="string"/>
+        <property name="date" type="string"/>
     </transfer>
 
     <transfer name="ComputopApiHeaderPayment">
-        <property name="payId" type="string" />
-        <property name="amount" type="int" />
-        <property name="transId" type="string" />
+        <property name="payId" type="string"/>
+        <property name="amount" type="int"/>
+        <property name="transId" type="string"/>
     </transfer>
 
     <transfer name="ComputopApiResponseHeader">
-        <property name="status" type="string" />
-        <property name="code" type="string" />
-        <property name="isSuccess" type="bool" />
-        <property name="description" type="string" />
-        <property name="transId" type="string" />
-        <property name="payId" type="string" />
-        <property name="xId" type="string" />
-        <property name="mId" type="string" />
-        <property name="mac" type="string" />
-        <property name="method" type="string" />
+        <property name="status" type="string"/>
+        <property name="code" type="string"/>
+        <property name="isSuccess" type="bool"/>
+        <property name="description" type="string"/>
+        <property name="transId" type="string"/>
+        <property name="payId" type="string"/>
+        <property name="xId" type="string"/>
+        <property name="mId" type="string"/>
+        <property name="mac" type="string"/>
+        <property name="method" type="string"/>
     </transfer>
 
     <transfer name="ComputopApiEasyCreditStatusResponse">
         <property name="header" type="ComputopApiResponseHeader"/>
-        <property name="refNr" type="string" />
-        <property name="Decision" type="string" />
-        <property name="Process" type="string" />
-        <property name="Financing" type="string" />
-        <property name="codeExt" type="string" />
-        <property name="ErrorText" type="string" />
+        <property name="refNr" type="string"/>
+        <property name="Decision" type="string"/>
```

Comment:
The name of the transfer fields must begin with a lowercase letter:
* `Decision` => `decision`.
* `Process` => `process`.
* `Financing` =>`financing`.
* `ErrorText` => `errorText`.